enterprise/frontend/src/metabase-enterprise/collections/index.ts

```diff
@@ -40,7 +40,7 @@ if (hasPremiumFeature("official_collections")) {
     if (isRegularCollection(collection)) {
       return [
         {
-          title: t`Make collection official`,
+          title: t`Make official`,
```

Comment:
I just noticed we have inconsistent Official/official casing here

![Screen Shot 2024-10-03 at 1 40 44 PM](https://github.com/user-attachments/assets/d9208597-bb38-40e1-8681-6f6e44b03de3) ![Screen Shot 2024-10-03 at 1 40 38 PM](https://github.com/user-attachments/assets/63a79180-8303-42fe-a771-e583632589a4)